torch/quantization/fx/quantization_patterns.py

```diff
@@ -67,6 +67,27 @@ def __init__(self, quantizer: QuantizerCls, node: Node):
         # all inputs are tensors or not, e.g. add/mul
         self.num_tensor_args = len(node.args)
         self.all_node_args_are_tensors = True
+        # the last node of the matched pattern
+        self.last_node = node
+
+    def _maybe_get_last_node_only_observer(
+        self,
+        quantizer: QuantizerCls,
+    ) -> Optional[Union[torch.quantization.ObserverBase, torch.quantization.FakeQuantizeBase]]:
+        """
+        If the last node of the pattern is observed, return the observer
+        instance. Otherwise, return None.
+        """
+        for maybe_obs_node, _ in self.last_node.users.items():
+            if maybe_obs_node.op == 'call_module':
+                maybe_obs = quantizer.modules[maybe_obs_node.name]
+                if (
```

Comment:
nit: we have: https://github.com/pytorch/pytorch/blob/master/torch/quantization/quantize.py#L23